react/ListPrice.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext } from 'react'
+import { defineMessages } from 'react-intl'
+import { ProductContext } from 'vtex.product-context'
+import { FormattedCurrency } from 'vtex.format-currency'
+import { useCssHandles } from 'vtex.css-handles'
+import { IOMessageWithMarkers } from 'vtex.native-types'
+
+import { StorefrontFC } from './types'
+
+interface Props {
+  label: string
+  markers: string[]
+}
+
+const CSS_HANDLES = ['listPrice', 'listPriceValue'] as const
+
+const ListPrice: StorefrontFC<Props> = props => {
+  const { label, markers } = props
+  const { selectedItem } = useContext(ProductContext)
+  const handles = useCssHandles(CSS_HANDLES)
+  const listPrice = selectedItem.sellers[0].commertialOffer.ListPrice
+
+  return (
+    <div className={handles.listPrice}>
+      <IOMessageWithMarkers
+        label={label}
+        markers={markers}
+        componentName={CSS_HANDLES[0]}
```

Comment:
Not sure if this makes much sense, semantically.
Why not just use `"productPrice"`?